api/contrib/envoy/extensions/private_key_providers/qat/v3alpha/qat.proto

```diff
@@ -0,0 +1,41 @@
+syntax = "proto3";
+
+package envoy.extensions.private_key_providers.qat.v3alpha;
+
+import "envoy/config/core/v3/base.proto";
+
+import "google/protobuf/duration.proto";
+
+import "udpa/annotations/sensitive.proto";
+import "udpa/annotations/status.proto";
+import "validate/validate.proto";
+
+option java_package = "io.envoyproxy.envoy.extensions.private_key_providers.qat.v3alpha";
+option java_outer_classname = "QatProto";
+option java_multiple_files = true;
+option go_package = "github.com/envoyproxy/go-control-plane/envoy/extensions/private_key_providers/qat/v3alpha";
+option (udpa.annotations.file_status).package_version_status = ACTIVE;
+
+// [#protodoc-title: `QAT` private key provider]
+// [#extension: envoy.tls.key_providers.qat]
+
+// This message specifies how the private key provider is configured.
+// The private key provider provides RSA sign and decrypt operation
+// hardware acceleration.
+
+// [#extension-category: envoy.tls.key_providers]
```

Comment:
Please remove this line as we do need one `envoy.tls.key_providers` category.

Reply:
Ok, fixed.

Reply:
Ping @daixiang0 if you are able to re-review and stamp